static/js/containers/ErrorPage_test.js

```diff
@@ -55,7 +55,7 @@ describe("ErrorPage", () => {
     [
       403,
       "You do not have permission to view this video",
-      "You do not have permission to view this video."
+      "Please contact course faculty or staff and ask them to add you to the Stellar course student or listener list."
```

Comment:
```suggestion
      "Please contact course faculty or staff and ask them to add you as a student or listener on stellar.mit.edu."
```

a student or listener on stellar.mit.edu

Reply:
@pdpinch I just updated the screenshot with the new text

Reply:
@pdpinch if the new screenshot looks good to you I will merge this